apps/docs/src/guide/cookbook/snippets/splitting-utxos.ts

```diff
@@ -0,0 +1,55 @@
+// #region splitting-utxos
+import { BN, Provider, Wallet, splitUTXOs } from 'fuels';
+
+import { LOCAL_NETWORK_URL, WALLET_PVT_KEY } from '../../../env';
+
+const provider = await Provider.create(LOCAL_NETWORK_URL);
+const fundingWallet = Wallet.fromPrivateKey(WALLET_PVT_KEY, provider);
+
+const wallet = Wallet.generate({ provider });
+
+// let's fund the wallet with 1000 of the base asset
+const fundingTx = await fundingWallet.transfer(
+  wallet.address,
+  1000,
+  provider.getBaseAssetId()
+);
+await fundingTx.waitForResult();
+
+// We can fetch the coins to see how many UTXOs we have, and confirm it is 1.
+const { coins: initialCoins } = await wallet.getCoins(
+  provider.getBaseAssetId()
+);
+console.log('Initial Coins Length', initialCoins.length);
+// 1
+
+// Now we can split the UTXO into 5 UTXOs of 200 each.
+const splitTx = splitUTXOs(
+  new BN(1000),
+  new BN(200),
+  provider.getBaseAssetId(),
+  wallet.address,
+  5
+);
+
+// We will also add some funds to the wallet to cover the fee
+const fundTx = await fundingWallet.transfer(
+  wallet.address,
+  500,
+  provider.getBaseAssetId()
+);
+await fundTx.waitForResult();
```

Comment:
I don't understand this part of the documentation. Why is the wallet being funded a second time?

Reply:
Because the wallet has a balance of 1000 comprised of 5 UTXOs each worth 200. Therefore there is an insufficient amount to cover the fee.

Reply:
Can we better clarify this in the comment where we fund the wallet the second time?